addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -260,7 +260,16 @@ export class MassMailingHtmlField extends HtmlField {
         const sidebar = document.querySelector("#oe_snippets");
         if (!sidebar) {
             return;
-        } else if (!this._isFullScreen()) {
+        } else if (this._isFullScreen()) {
+            sidebar.style.height = "";
+            sidebar.style.top = "0";
+        } else if (this.env.inDialog) {
+            const scrollableY = closestScrollableY(sidebar);
+            if (scrollableY) {
+                const rect = scrollableY.getBoundingClientRect();
+                sidebar.style.height = `${rect.height}px`;
```

Comment:
We never need to set the "top" style in this case?

Reply:
`.o_field_mass_mailing_html #oe_snippets` rules is by default at `top: 0` and opening in fullscreen puts `top: 0` anyway, so it does not really matter, but I can add it for consistency with the other conditions :)